### DISCUSSION TITLE
traceback.format_exc(e),  cant pass e, that will cause exception in exception then crash

### DISCUSSION THREAD

**Discussion by alenwesker** [microsoft/debugpy#1665] [Q&A]
Before creating a new issue, please check the [FAQ](https://github.com/microsoft/debugpy/wiki/FAQ) to see if your question is answered there.

## Environment data

- debugpy version: newest
- OS and version: any
- Python version (& distribution if applicable, e.g. Anaconda): any newer version
- Using VS Code or Visual Studio:

## Actual behavior

I was writing traceback.format_exc(e) in my project as well, as far as I remember, that's the way in the older versions, but any newer document and versions said, traceback.format_exe() does not accept e as the parameter

I am cleaning my code, then found that debugpy also contains lines like that: like eval_condition in breakpoint.py

Can anyone tell me when Python deletes that first parameter? Since there is code in pydebugy like that, we could call it like that once upon a time.

## Expected  behavior

cause exception in exception then crash

## Steps to reproduce:
```
import sys

def some_function():
    try:
        raise RuntimeError("Some error message")
    except Exception as e:
        import traceback
        sys.stderr.write(traceback.format_exc(e))

if __name__ == '__main__':
    some_function()
```

**Comment by rchiodo** [comment]
This isn't an issue with debugpy. Moving to discussions.

**Comment by rchiodo** [comment]
traceback changed somewhere along the way. If you want to do what you're doing here, you'd do this instead:

``` python
import sys

def some_function():
    try:
        raise RuntimeError("Some error message")
    except Exception as e:
        import traceback
        sys.stderr.write("\n".join(traceback.format_exception(e)))

if __name__ == '__main__':
    some_function()
```
